onnx/defs/shape_inference.h

```diff
@@ -44,17 +45,43 @@ inline bool hasExactlyNInputTypes(InferenceContext& ctx, int n, const std::strin
   return true;
 }
 
-inline void propagateElemTypeFromInputToOutput(InferenceContext& ctx, size_t inputIndex, size_t outputIndex) {
-  ctx.getOutputType(outputIndex)->set_elem_type(ctx.getInputType(inputIndex)->elem_type());
+inline void propagateElemTypeFromInputToOutput(
```

Comment:
Change name to "propagateTypeFromInputToOutput"?

Reply:
I found that what it was supposed to do is really coping element type instead of whole type proto.